ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -347,10 +348,46 @@ func (r *RayJobReconciler) Reconcile(ctx context.Context, request ctrl.Request)
 	case rayv1.JobDeploymentStatusComplete, rayv1.JobDeploymentStatusFailed:
 		// If this RayJob uses an existing RayCluster (i.e., ClusterSelector is set), we should not delete the RayCluster.
 		logger.Info(string(rayJobInstance.Status.JobDeploymentStatus), "RayJob", rayJobInstance.Name, "ShutdownAfterJobFinishes", rayJobInstance.Spec.ShutdownAfterJobFinishes, "ClusterSelector", rayJobInstance.Spec.ClusterSelector)
```

Comment:
Move the log message below and add the following 4 keys:

* `ttlSecondsAfterFinished`
* `Status.endTime`
* `Now`
* `ShutdownTime`

Then, remove these 4 keys from the following log messages.

Reply:
done